### DISCUSSION TITLE
"User Not Authorized" VSCodium Latest Student Copilot

### DISCUSSION THREAD

**Discussion by Tushar49** [community/community#19352] [Copilot Conversations]
I have been using CoPilot on VSCode by Microsoft since its beta version was released, and recently I got the student version for CoPilot once its free version was over. Today, to get rid of Telemetry, I shifted from VSCode to VSCodium and tried to log in to GitHub Extensions using a token access code. But CoPilot is throwing me an error :( telling me the user is not authorized. Please help me solve this; I cannot use it for the whole day.

**Comment by vedantmgoyal9** [comment]
Copilot  only works on VSCode and not on VSCodium. You have to switch to VSCode.

**Reply by Tushar49** [reply]
There must be some workaround since VSCodium is exactly as VSCode in build, with just Telemetry wholly removed. There might be something blocking it? Can it be fixed in the next update?

**Comment by Tushar49** [comment]
Every other extension of GitHub is working perfectly with VSCodium so far; even Github Desktop is well integrated with it. Surely there must be a similar solution for copilot, right? Please help me with this.

**Reply by priyanshivyas** [reply]
I am facing similar issues, urgent help needed.

**Comment by melvincwng** [comment]
@Tushar49, kindly follow the suggestions in this discussion thread over [here in #19361](https://github.com/github-community/community/discussions/19361).

Hopefully it helps to resolve the issues you are facing.

If it helps, kindly 'Mark as answer' so the rest can find the solution if they encounter the same issue as you.

**Reply by Tushar49** [reply]
I tried it too, even Github Copilot Nightly is throwing me the same error.
![image](https://user-images.githubusercontent.com/76201310/175787402-841fff60-23a7-44aa-967e-24cb16948c71.png)

**Reply by davecheney** [reply]
This sounds like a connectivity issue. 
Is it possible that your employer, internet service provider, or government, may be intercepting your SSL traffic?

**Reply by Tushar49** [reply]
No, it is impossible to have a connectivity issue since I am on the same network using the same ISP under the same government as I was under when I used VSCode. Also, I tried it with VPNs to ensure it's not a connectivity issue, and the results are identical. It is still giving me a "User not authorized" error.

**Comment by davecheney** [comment]
@Tushar49 vscodium is not currently supported. Please use vscode.

**Reply by Tushar49** [reply]
The question is why? And when will it be supported? Cause really there is no difference between those two except for the telemetry.

**Reply by davecheney** [reply]
I’m afraid I don’t know the answer, I don’t work on that team.

**Reply by Tushar49** [reply]
Its okay sir, thank you for your help! Someone must be there to solve this out, I will also see if I can compare the source codes of the 2 and check logs.
